### PR TITLE
Repository Reorganization: Directory Structure, License Consolidation, and File Cleanup

### PR DESCRIPTION
**Pull request by anvega**
This pull request includes the initial reorganization of the repository according the [Information Architecture ](https://docs.google.com/document/d/1bdcZ3WIBmLrTTZj_1aLp3fGl68WqEhifItIHThE53k8/edit)proposed, discussed, and approved by all leads to improve accessibility and maintainability. Key changes include:

* Created new directories: community/working-groups, community/events, and community/resources.
* Moved audio-versions to community/publications
* Moved  supply-chain-security, compliance, and controls content to community/working-groups subdirectories and merged readmes
* Moved policy to community/working-groups/archive
* Moved design, security-lexicon, security-whitepaper, landscape, security-fuzzing-handbook to community/resources
* Moved cloud_native_security.md, and past-events.md to community/events
* Deleted unused files: policy-wg-merging.md, roadmap.md, safe_kubecon.md.
* Updated relative paths for links 
* Updated a few deadlinks a different markdown files

### CODE REVIEW

File: LICENSE.md
Comment (eddie-knight) [inline LICENSE.md:1]: I have thought about this in recent weeks, but wasn't sure where this gets into CNCF legal territory.

@TheFoxAtWork, do you have any insight to add as a previous contributor to these files?

File: community/catalog/README.md
Comment (eddie-knight) [inline community/catalog/README.md:1]: Moving the compromises makes sense. Do we have a plan to put anything else in the `catalog` directory? Should we reduce one dir layer here?

File: LICENSE.md
Comment (anvega) [reply LICENSE.md:1]: What's the question? To simplify the repository structure and provide clarity on licensing terms, the three existing license files (LICENSE-code, LICENSE-document, LICENSE.md) were merged into a single LICENSE.md file. To be clear there are no licensing changes made here. The updated license file covesr both code and documentation, as follows:

```
LICENSE.md

# License

## Source Code

The source code in this repository is licensed under the MIT License. See below for the full license text.

## Documentation

The documentation in this repository is licensed under the Creative Commons Attribution 4.0 International License. See below for the full license text.

### MIT License

[Full text of the MIT]

### Creative Commons Attribution 4.0 International License

[Full text of the CC BY 4.0 License]

```

Comment (anvega) [reply LICENSE.md:1]: cc/ @mkdolan

Comment (eddie-knight) [reply LICENSE.md:1]: Yep, just looking for input from anyone else who can confirm what I agree with you on— merging these files is a good way to consolidate the presence of both licenses.

File: community/catalog/README.md
Comment (anvega) [reply community/catalog/README.md:1]: Yes, the goal is to periodically generate reports, and those scripts, along with the static PDFs, would go into this directory. The catalog directory is meant to include assets related to the catalog initiative itself, rather than the entries of compromises or their content. Therefore, notes and discussions about the catalog will be placed in the catalog directory, not directly inside the compromises subdirectory.

Comment (eddie-knight) [reply community/catalog/README.md:1]: Is that captured anywhere? Doesn't need to happen within this PR, but we might add it to the pile of things to document.

Comment (anvega) [reply community/catalog/README.md:1]: Yes, Information Architecture document captures this as it models catalog as a working group. Same doc explained the licensing change too.

File: community/catalog/compromises/2020/thegreatsuspender.md
Comment (eddie-knight) [inline community/catalog/compromises/2020/thegreatsuspender.md:26]: What's happening with this word change? It seems like an impactful tweak, considering that it shifts from discussing the policy for the linked repo to implying that many policies may exist. Better to leave this for a suggested change that the WG can respond to.

Comment (anvega) [reply community/catalog/compromises/2020/thegreatsuspender.md:26]: The page doesn't exist anymore and the repository doesn't have a privacy policy. I adjusted enough to say there was a policy but that policy isn't *the* policy.

Comment (anvega) [reply community/catalog/compromises/2020/thegreatsuspender.md:26]: This effort is part of addressing your feedback highlighted during the approval discussion of the IA document, specifically regarding 'broken links throughout the repo that we'll need to find and clean up,' but fine by me either way. 

The wording probably should be a 'privacy policy that existed for a period of time' or 'at the time.' I tried Wayback Machine to replace it with that but it wasn't able to pull it

Comment (eddie-knight) [reply community/catalog/compromises/2020/thegreatsuspender.md:26]: Ah, frick. I see how you got there. 

Can we get a thumbs up from the WG lead or last committer? If we don't hear back, I'd suggest its better to leave with a broken link than update it with a commit that may not improve the doc.

File: LICENSE.md
Comment (anvega) [reply LICENSE.md:1]: You mentioned CNCF legal, so I tagged them. Also, @joannalee333 can help with a thumbs up here.

There are several online resources on multi-licensing and declaring those multiple licenses. One of GitHub’s support recommendations is to use one LICENSE file in the root of your repository and add subsections for any 3rd party material, code, etc. which is the approach being pursued here. 

An improvement complementary to this approach is to clearly mention the terms in the `README.md`:

`The source code in this repository is licensed under the MIT License.
The documentation is licensed under the Creative Commons Attribution 4.0 International License.
`

File: compliance/README.md
Comment (anvega) [reply compliance/README.md:1]: Yes, this is already merged at `community/compliance/README.md` consistent with the structure and format of the markdown files of other working groups.

File: community/catalog/compromises/compromise-definitions.md
Comment (eddie-knight) [inline community/catalog/compromises/compromise-definitions.md:1]: This doesn't match the link in the README above, which would place this (perhaps rightfully) as `catalog/compromise-definitions.md`

File: community/catalog/compromises/README.md
Comment (eddie-knight) [inline community/catalog/compromises/README.md:26]: see comment on `catalog/compromises/compromise-definitions.md`

**Comment by PushkarJ** [comment]
Looks like this needs a rebase + resolving of merge conflicts

File: community/publications/README.md
Comment (eddie-knight) [inline community/publications/README.md:1]: Do we have enough of a consensus on this proposal to nest publications within `community/`? An alternate proposal was to have publications as a top level directory. 

I had been fond of the idea of publications living at the top, but I can also see the justification for nesting it here.

@mnm678 @PushkarJ @mlieberman85 @JustinCappos @ragashreeshekar @ashutosh-narkar @jkjell

File: community/resources/security-whitepaper/cnsmap/README.md
Comment (eddie-knight) [inline community/resources/security-whitepaper/cnsmap/README.md:1]: Should security whitepaper be categorized as a publication or community resource?

File: community/working-groups/supply-chain-security/README.md
Comment (eddie-knight) [inline community/working-groups/supply-chain-security/README.md:1]: @mnm678 @mlieberman85 @jkjell Please review

File: community/working-groups/supply-chain-security/secure-software-factory/Secure_Software_Factory_Whitepaper.pdf
Comment (eddie-knight) [inline community/working-groups/supply-chain-security/secure-software-factory/Secure_Software_Factory_Whitepaper.pdf:1]: @PushkarJ @mnm678 — Since we're already breaking links, do we want to take this opportunity to migrate publications like this into the publications directory?

File: policy-wg-merging.md
Comment (eddie-knight) [inline policy-wg-merging.md:1]: Should this be moved into the wg/archive/policy directory?

File: roadmap.md
Comment (eddie-knight) [inline roadmap.md:1]: @mnm678 @PushkarJ Pleaes review. 

I agree that this file is less than ideal, but please verify we haven't lost anything by deleting it.

File: community/publications/README.md
Comment (anvega) [reply community/publications/README.md:1]: There is list of select publications in the top level README.md and a link to the directory. The publications directory doesn't hold any publications, only an index and the directory of audio recordings.

File: community/resources/security-whitepaper/cnsmap/README.md
Comment (anvega) [reply community/resources/security-whitepaper/cnsmap/README.md:1]: Everything is listed in the publications page. As stated in the IA document, overarching/seminal/tag-wide publications that aren't owned by a workgroup are to be found there as valuable community resources.

File: community/working-groups/supply-chain-security/secure-software-factory/Secure_Software_Factory_Whitepaper.pdf
Comment (anvega) [reply community/working-groups/supply-chain-security/secure-software-factory/Secure_Software_Factory_Whitepaper.pdf:1]: The consensus achieved through the IA document discussions settled on the structure captured there:

> **community/publications/**
> Index of published papers, reports, whitepapers, and audio versions of publications. The source files are located in:
> * community/resources/ if they are overarching documents.
> * community/working-groups/ if they are owned by a working group.
> * communityassessments/ if they are assessment documents.

If you're proposing something different, we should discuss and plan for it after this current reorganization is complete.

File: community/resources/security-whitepaper/cnsmap/README.md
Comment (anvega) [reply community/resources/security-whitepaper/cnsmap/README.md:1]: Here is the structure that the group consensus settled on:

> community/publications/
> Index of published papers, reports, whitepapers, and audio versions of publications. The source files are located in:
> 
> community/resources/ if they are overarching documents.
> community/working-groups/ if they are owned by a working group.
> communityassessments/ if they are assessment documents.

File: community/publications/README.md
Comment (anvega) [reply community/publications/README.md:1]: I have no recollection of the alternate proposal being discussed. The only other discussion involved this PR that was merged, which reinforced the list of selected publications in the `README.md` and provided an index of source files with their respective locations inside `publications/README.md` (#1273).

File: policy-wg-merging.md
Comment (anvega) [reply policy-wg-merging.md:1]: None of the key deliverables stated in that file were turned in. Also the Policy workgroup never really merged with TAG security and stayed they stayed their course as a Kubernetes WG.

File: roadmap.md
Comment (anvega) [reply roadmap.md:1]: I think this predates all of you. We adopted project boards as captured in this markdown file and migrated everything worth tracking to the board and the leadership notes doc. I don't believe there is anything relevant in the old roadmap other than nostalgic value.

This was captured in the doc as an item marked for deletion.

File: policy-wg-merging.md
Comment (anvega) [reply policy-wg-merging.md:1]: This was a file captured in the doc as marked for deletion not move.

File: community/working-groups/supply-chain-security/README.md
Comment (anvega) [reply community/working-groups/supply-chain-security/README.md:1]: For reviewers this was a merge the contents with the prior README.md,

File: community/catalog/compromises/compromise-definitions.md
Comment (anvega) [reply community/catalog/compromises/compromise-definitions.md:1]: It's unclear what's meant by `./compromise-definitions.md` not matching its link in `compromises/README.md`, as both of these files reside in the same directory. The link is working, and the dot is correctly used as the symbol for the current directory.

By README above you mean `community/catalog/compromises/README.md` or `compromises/README.md`?

Comment (anvega) [reply community/catalog/compromises/compromise-definitions.md:1]: Line 11 did have a broken link I failed to catch as was truncated by a break. It pointed to `/supply-chain-security/compromises/compromise-definitions.md` when it should be `community/catalog/compromises/compromise-definitions.m`

File: LICENSE.md
Comment (anvega) [reply LICENSE.md:1]: Heard back from CNCF legal. They said on 6/24: 

> Hi Andres, I reviewed. From a purely legal perspective, I have no concerns. I want to check with someone on the team here thought to make sure the change will still work with license header scanning tools used by CNCF projects.

I don't know that TAG folders are scanned, but there seems to be no issue here.

**Review by mnm678 — COMMENTED** [review]
This lgtm after a rebase and fixing the failing checks

**Comment by anvega** [comment]
Yay! I don't know what changed, but suddenly deploy previews and site build logs became visible once again. The preview error was due to a malformed URL in the Spanish translation of the whitepaper (cloud-native-security-whitepaper-spanish.md). Site is functional. Rebased and a good to go.

**Review by mnm678 — APPROVED** [review]

**Comment by anvega** [comment]
@eddie-knight Are you comfortable with the current state before merging? There was extensive commentary, but it wasn’t clear if the feedback required changes. Please let me know if there's anything else you'd like to see addressed.

**Comment by anvega** [comment]
Merging as discussed in the chair and TLs' communications channel, as no concerns were raised following the final review request earlier.

**Pull request merged**
